Source/Interfaces/Preferences.xib

```diff
@@ -1140,14 +1140,34 @@ Gw
                         <binding destination="117" name="value" keyPath="values.EditInSheetForMultiLineText" id="mbU-Zr-d1A"/>
                     </connections>
                 </button>
+                <button verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="YH2-qI-BBK">
+                    <rect key="frame" x="178" y="14" width="218" height="18"/>
+                    <buttonCell key="cell" type="check" title="Sort descending by primary key" bezelStyle="regularSquare" imagePosition="left" alignment="left" inset="2" id="L4y-ub-YHv">
+                        <behavior key="behavior" changeContents="YES" doesNotDimImage="YES" lightByContents="YES"/>
+                        <font key="font" metaFont="system"/>
+                    </buttonCell>
+                    <connections>
+                        <binding destination="117" name="value" keyPath="values.SortDescendingByPrimaryKey" id="NWV-5B-LN1"/>
+                    </connections>
+                </button>
```

Comment:
I definitely agree with the comment about moving to a dropdown instead of a checkbox!